demos/checkbox.html

```diff
@@ -201,6 +202,10 @@
         document.getElementById('make-ind').addEventListener('click', function() {
           checkboxInstance.indeterminate = true;
         });
+
+        document.getElementById('toggle-disabled').addEventListener('click', function() {
```

Comment:
This should also add the correct `mdc-checkbox--disabled` modifier class to the example `mdc-checkbox` so the hover state behavior is correct.